lib/negroku/templates/tasks/eye/eye.rb.erb

```diff
@@ -10,8 +10,8 @@ Eye.application '<%= fetch(:application)%>' do
   # except `env`, which merging down
 
   working_dir File.expand_path(File.join(File.dirname(__FILE__), %w[ processes ]))
```

Comment:
I'm not sure because the current path changes on deploy. I'don't know what would happen to the eye monitor when its working dir changes on runtime. Maybe I'm thinking it wrong?